neutone_sdk/utils.py

```diff
@@ -169,6 +169,15 @@ def save_neutone_model(
             f"{loaded_model.calc_model_delay_samples()}"
         )
 
+        log.info("Testing offline mode...")
+        for input_sample in get_default_audio_samples():
+            offline_sr = input_sample.sr
+            offline_bs = 4096
+            loaded_model.set_daw_sample_rate_and_buffer_size(offline_sr, offline_bs)
+            offline_params = tr.rand((MAX_N_PARAMS, input_sample.audio.size(1)))
+            offline_rendered_sample = loaded_model.forward_offline(input_sample.audio, offline_params)
+            break
```

Comment:
Remove this

Reply:
I was thinking doing all 3 samples would make the export unnecessarily long without providing additional testing insight since for bigger models it could take some time to complete. I can add a comment to the break statement clarifying this.

Reply:
Oh, I see. Maybe you can just run it on get_default_audio_samples()[0], but this is also fine.